.gitignore

```diff
@@ -0,0 +1,107 @@
+# Mac files
```

Comment:
I'd look to keep this minimal. A lot of these entries are specific to tools we're unlikely to use anyway. Something like [this](https://github.com/companieshouse/kafka-stack/blob/main/groups/zookeeper/.gitignore). We can add as needed